packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -159,3 +159,17 @@ class TapSemanticEvent extends SemanticsEvent {
   @override
   Map<String, dynamic> getDataMap() => const <String, dynamic>{};
 }
+
+/// An event which triggers focus semantic.
+///
+/// This is generally not recommended to use, it may break the consistency of the
+/// accessibiliy focus of the platform.
```

Comment:
It might be helpful to give an example of when this would be used and example usage.